frontend/test/metabase-smoketest/user.cy.spec.js

```diff
@@ -6,10 +6,7 @@ describe("smoketest > user", () => {
   beforeEach(signInAsNormalUser);
 
   it("should be able to ask a custom questions", () => {
-    cy.visit("/");
-
-    cy.findByText("Ask a question").click();
-    cy.findByText("Simple question");
```

Comment:
I think this should be left in, since it's a smoketest and should follow the regular user flow, not direct URLs.

Reply:
Agreed.
Since this has been already merged in master, I'll leave it as it is and make a note for myself to change it back when I start refactoring smoke tests.